sbi2-2023-cytotable-poster/poster.html

```diff
@@ -0,0 +1,589 @@
+width: 85%;
+margin-left: auto;
+margin-right: auto;
+}
+code.sourceCode.r, code.sourceCode.python {
+background-color: transparent;
+font-size: 20pt;
+border-radius: 2mm;
+}
+code {
+font-size: 25pt;
+font-family: monospace;
+background-color: #FED7AA24;
+color: #FED7AA;
+padding: 1.2mm;
+line-height: 1;
+border-radius: 2mm;
+}
+.caption {
+font-size: 20pt;
+}
+.table caption {
+font-size: 20pt;
+padding-bottom: 3mm;
+
+}
+code {
+font-size: 1em;
+font-family: monospace;
+background-color: #FED7AA24;
+color: #FDE68A;
+padding: 1.2mm;
+border-radius: 2mm;
+}
+.poster_title code {
+font-size: 1em;
+}
+table {
+font-size: 40px;
+margin: auto;
+border-top: 3px solid #666;
+border-bottom: 3px solid #666;
+}
+table thead th {
+border-bottom: 3px solid #ddd;
+}
+td {
+padding: 8px;
+}
+th {
+padding: 15px;
+}
+caption {
+margin-bottom: 10px;
+}
+.poster_body p {
+margin-right: 4mm;
+margin-left: 4mm;
+margin-top: 6mm;
+margin-bottom: 10mm;
+color: #000000
+}
+.poster_body ol {
+margin-right: 4mm;
+margin-left: 4mm;
+}
+#ul {
+margin-right: 4mm;
+margin-left: 4mm;
+}
+.references p {
+font-size: 20pt;
+}
+.orcid img {
+  width: 1em;
+}
+</style>
+</head>
+<body>
+
+
+<div class="poster_wrap">
+<div class="title_container">
+<!-- Left Logo  -->
+<div class="logo_left">
+<img src=images/dbmi.png style="width: 80%">
+</div>
+<!-- Poster Title -->
+<div class= "poster_title">
+<h1 id="title">CytoTable: High Performance and Scalable <br>Single-cell Morphology Feature Engineering</h1>
+<h3 id="author">Dave Bunten<sup>1, <a class="orcid" href="https://orcid.org/0000-0001-6041-3665"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup>, Erik Serrano<sup>1, <a class="orcid" href="https://orcid.org/0000-0003-4188-9080"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup>, Jenna Tomkinson<sup>1, <a class="orcid" href="https://orcid.org/0000-0003-2676-5813"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup>, Michael J. Lippincott<sup>1, <a class="orcid" href="https://orcid.org/0000-0002-8637-1448"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup>, Faisal Alquaddoomi<sup>1, <a class="orcid" href="https://orcid.org/0000-0003-4297-8747"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup>, Vince Rubinetti<sup>1, <a class="orcid" href="https://orcid.org/0000-0002-4655-3773"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup>, Gregory P. Way<sup>1, <a class="orcid" href="https://orcid.org/0000-0002-0503-9348"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup></h3>
+<h5 id="affiliation"><sup>1</sup> Department of Biomedical Informatics, University of Colorado Anschutz</h5>
+</div>
+<!-- Right Logo  -->
```

Comment:
Would it be possible to add the cytomining logo next to the qr code mixed with the cytotable logo?

Reply:
Thanks - yes, I've added the Cytomining logo next to the CytoTable with a QR code 🙂